core/src/main/java/com/netflix/iceberg/TableOperations.java

```diff
@@ -56,27 +55,15 @@
   void commit(TableMetadata base, TableMetadata metadata);
 
   /**
-   * Create a new {@link InputFile} for a path.
-   *
-   * @param path a string file path
-   * @return an InputFile instance for the path
+   * Obtain a handler to read and write files.
```

Comment:
This should note that it returns a FileIO object and link to the javadoc for FileIO.